### PR TITLE
Added in output disabling

### PR DESCRIPTION
**Pull request by Luminarys**

### CODE REVIEW

**Comment by SirCmpwn** [comment]
Update the man page.

**Comment by Luminarys** [comment]
Should I change it so that it reads `<disable|resolution|res...` or `<disable>|<resolution|res...` ?

**Comment by SirCmpwn** [comment]
Make it a second point, [like this](https://github.com/SirCmpwn/sway/blob/master/sway.5.txt#L49-L57).

**Comment by Luminarys** [comment]
Alright, should be good.

File: sway/commands.c [hunk 1]
```diff
@@ -372,8 +372,13 @@ static bool cmd_output(struct sway_config *config, int argc, char **argv) {
 	struct output_config *output = calloc(1, sizeof(struct output_config));
 	output->x = output->y = output->width = output->height = -1;
 	output->name = strdup(argv[0]);
+	output->enabled = true;
 
 	// TODO: atoi doesn't handle invalid numbers
+	
+	if (strcmp(argv[1], "disable") == 0) {
```
Comment (SirCmpwn) [inline sway/commands.c]: We use strcasecmp for almost everything, let's continue that pattern.

File: sway/handlers.c [hunk 2]
```diff
@@ -104,6 +108,8 @@ static void handle_output_destroyed(wlc_handle output) {
 	}
 	if (i < list->length) {
 		destroy_output(list->items[i]);
+	} else {
+		return;
```
Comment (SirCmpwn) [inline sway/handlers.c]: Return a boolean here.

Comment (Luminarys) [inline sway/handlers.c]: Isn't the return type void?

Comment (SirCmpwn) [inline sway/handlers.c]: Sorry, didn't expand the diff. Thought this was part of `handle_output_created`.

**Pull request merged**